fe/fe-core/src/main/java/org/apache/doris/qe/ShowExecutor.java

```diff
@@ -977,18 +977,25 @@ private void handleShowLoad() throws AnalysisException {
             ErrorReport.reportAnalysisException(ErrorCode.ERR_BAD_DB_ERROR, showStmt.getDbName());
         }
         long dbId = db.getId();
+        boolean isAccurateMatch = showStmt.isAccurateMatch();
+        String label = showStmt.getLabelValue();
+        if (!isAccurateMatch && label != null) {
+            if (!label.contains("%")) {
```

Comment:
This should be done in analyze() of ShowLoadStmt.

Reply:
done